scoper.inc.php

```diff
@@ -24,6 +24,7 @@
 		'SearchConsole',
 		'SiteVerification',
 		'TagManager',
+		'AnalyticsData',
```

Comment:
Nit: this list is alpha sorted, the order isn't significant other than for readability.

Reply:
Oops, noted, thank you!